storefront-reference-architecture-master/cartridges/int_twilio/cartridge/scripts/helpers/productSubscriptionHelper.js

```diff
@@ -0,0 +1,155 @@
+"use strict";
+var CustomObjectMgr = require("dw/object/CustomObjectMgr");
+var Calendar = require("dw/util/Calendar");
+var PHONE_NUMBERS_TABLE_NAME = "PRODUCT_SUBSCRIPTION";
+var PHONE_VERIFICATION_TABLE_NAME = "PHONE_VERIFICATION";
+
+const isExisting = (productId, phoneNumber) => {
+    const response = {};
+    const numberResult = CustomObjectMgr.getCustomObject(
```

Comment:
WIthin the back-end JS please use just var otherwise the debugger won't catch those

Reply:
fixed